### PR TITLE
Implement `sfw` query parameter.

### PR DESCRIPTION
**Pull request by darthshittious**
This implements filtering out NSFW words in quotes when the `sfw` query
parameter is passed, fixing #11. In this commit I have only filtered out words
containing some form of the word "fuck".

### CODE REVIEW

**Comment by JohannesNE** [comment]
Thank you for working on this!

There is a slight problem with the implementation.
Some minutes have only a NSFW quote:

With the current list these are:
```
litclock %>% filter(sfw == "yes") %>% anti_join(all_timestamps, .)
Joining, by = "time"
# A tibble: 7 × 1
  time 
  <chr>
1 00:05
2 04:59
3 09:25
4 09:26
5 09:42
6 09:43
7 09:44
```

A fix, that would also improve the code in general, would be to drop the duplication of quotes into several json files: https://github.com/JohannesNE/literature-clock/issues/17

Then the javascript would just try to load a json file, and check if a SFW quote exists. If either fails, it will try one minute earlier... and so on.

Are you up for fixing both?

File: docs/index.html [hunk 1]
```diff
@@ -6,12 +6,20 @@
...
-        function loadFile(time) {
+        function loadFile(time, sfw) {
             var timeStamp = timeToTimecode(time);
             return fetch("times/" + timeStamp + ".json")
                 .then(response => {
                     if (response.ok) {
-                        return response.json();
+                        var json = response.json();
+                        if (sfw) {
+                            json = json.filter(item => item.sfw === 'yes');
```
Comment (JohannesNE) [inline docs/index.html:16]: I get an error here.

Comment (darthshittious) [reply docs/index.html]: resolved in 23eeb75

**Comment by JohannesNE** [comment]
Nice! Thanks, would you mind adding a line about it to the README?

**Pull request merged**